test/common_utils.py

```diff
@@ -492,6 +492,34 @@ def get_num_channels(color_space):
     return num_channels
 
 
+def make_image(
+    spatial_size,
+    *,
+    color_space="RGB",
+    batch_dims=(),
+    dtype=torch.float32,
```

Comment:
In practice, transforms largely operate on `uint8` images as they come straight from PIL. Should this be the default instead?

Reply:
Sure, fine with me.